.chloggen/fix_apache-instrumentation-with-lifecycle.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: auto-instrumentation
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Apache instrumentation sidecar needs to remove lifecycle as it's not supported
```

Comment:
Please replace the sentence with a description of what you are fixing instead of the reason why you are doing it.